src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name=demo
```

Comment:
as name please put something meaningful here as name of the application

Reply:
done